packages/core/src/template-asciidoctor-node.js

```diff
@@ -28,5 +31,40 @@ export default function (moduleConfig) {
   Asciidoctor.prototype.getVersion = function () {
     return ASCIIDOCTOR_JS_VERSION
   }
+
+  // Alias
+  Opal.Asciidoctor.Cache = {
+    disable: function () {
+      // QUESTION: should we also reset cache?
+      const openUriSingleton = Opal.OpenURI.$singleton_class()
+      if (Opal.OpenURI['$respond_to?']('original_open_uri')) {
+        openUriSingleton.$send('remove_method', 'open_uri')
+        openUriSingleton.$send('alias_method', 'open_uri', 'original_open_uri')
+      }
+    },
+    reset: function() {
```

Comment:
We might rename this function to `clear`. That way it's "clear" (pun intended) that the function will clear the cache (and not reset the max value)